src/screens/Login/Start/StartContainer.js

```diff
@@ -16,14 +17,15 @@ const mapStateToProps = (state) => {
   }
 }
 
-const mapDispatchToProps = (dispatch) => {
-  return {
-    navigateToImportWallet: () => { },
-  }
-}
-/* eslint-enable no-unused-vars */
+const mapDispatchToProps = (dispatch) => bindActionCreators({}, dispatch)
 
 class StartContainer extends PureComponent {
+  constructor (props) {
+    super(props)
+    this.state={
+      showAccountsList: props.accounts && props.accounts.length !== 0 ? true : false,
```

Comment:
Do we need to reset state on component mount/unmount?

Reply:
I think we dont have to. Because user can accidentally go to other page and  then go back. For now he will have data saved